### PR TITLE
Link to Beeceptor updated

### PR DESCRIPTION
**Pull request by ankitjaininfo**
* Link to Beeceptor updated
* Some other grammatical changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ A lot of new tools have been developed since the list started, and many tools ha
 
 # Recommendations
 
-* For most people, I currently recommend [CloudFlare Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/). Although it's closed source, this is the production-quality service that gets the closest to achieving the dream. It's also a loss-leader for CloudFlare's other products which means they can offer it for free.
+* For most people, I currently recommend [CloudFlare Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/). Although it's a closed source, this is the production-quality service that gets the closest to achieving the dream. It's also a loss-leader for CloudFlare's other products which means they can offer it for free.
```
Comment (anderspitman) [inline README.md:33]: The previous wording sounds more natural to me. "Closed source" is often used as an adjective in my experience.

**Review by anderspitman — COMMENTED** [review]
Hi @ankitjaininfo, thank you for the improvements! Just one nit then I'd be happy to merge.

File: README.md (hunk 1 above)
Comment (ankitjaininfo) [reply README.md]: Addressed. Thanks for the review.

**Comment by anderspitman** [comment]
Looks good; thanks again!

**Pull request merged**